ui-lib/src/components/Layout/SideBar/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./SideBar";
```

Comment:
Any reason to not write the components directly inside `index.tsx` instead of re-exporting?